src/main/scripts/org/reaktivity/specification/nukleus/socks/streams/forward/client.connect.send.data.throttling.client.smaller/client.rpt

```diff
@@ -0,0 +1,47 @@
ongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010LongText110000000010"
```

Comment:
suggest writing a simple function to write this data, https://github.com/reaktivity/nukleus-http-cache.spec/blob/develop/src/main/java/org/reaktivity/specification/nukleus/http_cache/specification/internal/Functions.java#L113